src/test/java/org/candlepin/test/DatabaseTestFixture.java

```diff
@@ -650,6 +651,20 @@ protected Owner createOwner(String key, String name) {
         return owner;
     }
 
+    protected Owner createOwnerEnt(String key) {
```

Comment:
nack

If this is going to be a generalized test util method, it needs to be named better or have some docs associated with it. "owner ent" is ambiguous in terms of what it will actually do; especially within the context of some of the other testutil methods which indeed do things like create an owner with an entitlement.

Reply:
done